### DISCUSSION TITLE
Ability to hide shortcuts list on the rundown webpage window ?

### DISCUSSION THREAD

**Discussion by johnman51** [cpvalente/ontime#989] [Ideas]
Hello !
Again, Thank you for this beautiful project! I love to use it!
I'd like to know if it would be possible to add the ability to hide shortcuts list (on the right side) on the rundown webpage window ?
I can't shrink the window to hide it, it always resizes keeping the shortcut list visible…

Kind regards!
John.

**Comment by cpvalente** [comment]
Hi @johnman51 , thank you for your kind words and thank you for reaching out

The panel is a placeholder for editing an event. While there are no events selected it shows a list of shortcuts
There is currently no way to hide this and I am unsure I understand the use case.

Would you like to be able to hide the element altogether or just the shortcuts?
What are you attempting to gain? Are the shortcuts distracting or do you want some screen real-estate back?

**Comment by johnman51** [comment]
Hi, I'd just like to hide the shortcuts List to get the same view a it was in the V2 when I shrink the window of the webpage, here is the comparison in jpg…

Kind regards…
John.

![Comparison](https://github.com/cpvalente/ontime/assets/43176286/e95e2369-88ef-4ddc-92b2-2375a5e51c9c)

**Comment by johnman51** [comment]
Hi!
Thank you for having considering my request on the last update!
I still have the same issue with safari and the Better touch tool integrated web browser…
Actually, I use the rundown on the left side of a mixed quad view into Better Touch Tool software when I press a custom button… 

At least the best option I could imagine would be an option to have the ability to only view the Rundown List without the edit options on the right side…

I use this possibility as a Live option, considering that if I have to edit the cues deeper, I go back to the main Ontime full view…

I give you a view of what it looked like with the V2 version compared with the V3...

Do you think it could be possible?

Kind regards!
John

![Comparison V2vsV3](https://github.com/cpvalente/ontime/assets/43176286/4c657509-927d-4146-a5c9-ec8256c84816)

**Reply by cpvalente** [reply]
Hi @johnman51 , I believe this is already possible. can you change the panel mode to run?

**Reply by johnman51** [reply]
Unfortunately, it only works on Chrome Browser which can't be integrated into my solution...
(Settings are the same between V2 & V3 concerning the display settings on the left side of my snapshots, I don't have the ability to crop the window itself…)

**Reply by cpvalente** [reply]
hmm, I am slightly confused as to what is happening here.
If you click the Run mode, you signal to the app that you dont plan on editing and we close the panel.

That shouldn't be browser dependent, but maybe there is something about BTT that is making this misbehave?
Do you have a proposal for us on how we could solve this?

You also mentioned safari, but I seem to be able to get this to work fine here?
Perhaps I am misunderstanding the issue?
![run mode](https://github.com/cpvalente/ontime/assets/34649812/c5180196-0d2a-405a-8ebd-0fb82434bba3)

**Reply by johnman51** [reply]
Hi !
I feel terribly confused!

I really didn't see these two buttons! My Bad!!!
It's totally OK now!!!! (and in Safari too !!!!)

THANK YOU FOR THIS QUICK VIDEO TUTORIAL !!!!

Again, your project is Wonderfull!

Kind regards,
John.

**Reply by cpvalente** [reply]
thank you! I apologise for the confusion
